cobra/solvers/coin.py

```diff
@@ -27,7 +27,7 @@ def objectiveValue_(self):
 
 
 def create_problem(cobra_model, objective_sense="maximize", **kwargs):
-    m = cobra_model.to_array_based_model()
+    m = cobra_model.copy().to_array_based_model()
```

Comment:
Yeah, that's a bug. It's annoying and I'm not entirely sure how to fix it.

For now this is fine.